backend/pkg/api/connect/service/transform/mapper.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package transform
+
+import (
+	"fmt"
+	v1alpha1 "github.com/redpanda-data/console/backend/pkg/protogen/redpanda/api/dataplane/v1alpha1"
+)
+
+func intToInt32(i int) int32 {
```

Comment:
maybe I am not getting it, but do we really need this function?

Reply:
nope. I thought I needed a pointer initially, realized I didn't and then just didn't bother cleaning it up at the time. gone now!